GOVERNANCE.md

```diff
@@ -1,114 +1,185 @@
+* The person in question for approvership can nominate themselves, or be nominated by someone else.
+	* A contributor is nominated by someone submitting a PR to add their github handle to the approvers section of the [OWNERS](OWNERS.md) file.
+* To become an approver, a simple majority vote among the current approvers should be performed.
+    * See [conflict resolution](#conflict-resolution-and-voting) for more information on how to handle disputes.
 
-* Approvers have all of the responsibilities of the reviewers, as well as:
+#### Approver responsibilities
+
+Approvers have all of the responsibilities of the reviewers, as well as:
 * The expectation of reviews is higher. Approvers should be experts of their knowledge domain, and should serve as code owners of those pieces.
 * Participate when called upon in the [security release process](https://github.com/cri-o/cri-o/blob/main/SECURITY.md). Note that although this should be a rare occurrence, if a serious vulnerability is found, the process may take up to several full days of work to implement. This reality should be taken into account when discussing time commitment obligations with employers.
 
-## Emeritus Approver
+#### Distinction between top-level approvers and sub-tree approvers
+Through the mechanism of the OWNERS file, it’s possible to define sub-tree approvers for areas of specialty. For the purposes of this document, when “approvers” is mentioned, sub-tree approvers are not automatically included, though they could be. One way this happens is if a section of code that a sub-tree approver is responsible for has a security vulnerability, they will be included in the discussions. However, they need not necessarily be included if there is not such a vulnerability.
+
+### Emeritus Approver
 
 * All good things must come to an end, and an approver's tenure is no different.
 * The CRI-O community has included the emeritus approver title for approvers who have completed their service to the community, and have moved on to other things.
 * While the emeritus approver has no more power within the project, they do have wisdom and can be called upon for guidance.
 * If an approver has been inactive in the community (no pull requests or reviews) for more than 2 years, then they may be asked by the approvers to move themselves to an emeritus approver. This is to open up slots for new approvers to fill their slots.
 * To become an emeritus approver, one simply has to move themselves from the approvers field to the emeritus approvers field in the OWNERS file.
 
-## Demotion
-* In order to maintain an active reviewers and approvers list, the CRI-O team has a right to demote an approver or reviewer if they appear inactive for one year.
+### Demotion
+
+* In order to maintain an active reviewers and approvers list, the CRI-O team has a right to demote an approver or reviewer if they appear inactive for two years.
```

Comment:
two years seems too long.  I'd go a year, maybe 6 months.  YMMV

Reply:
I will drop to one